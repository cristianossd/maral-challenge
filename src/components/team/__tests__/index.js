import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Team from '../';

const teamMock = {
  name: 'Team mock',
  finalScore: 2,
  events: [
    { order: 1, time: '04:20', ranking: 1 },
  ],
};

describe('<Team />', () => {
  it('render correctly', () => {
    const wrapper = shallow(
      <Team
        attributes={teamMock}
        pos={1}
        hidden={false}
      />
    );

    expect(wrapper.find('tr').length).toBe(1);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
