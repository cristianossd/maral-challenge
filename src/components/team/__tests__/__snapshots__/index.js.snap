// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Team /> render correctly 1`] = `
<tr>
  <th
    scope="row"
  >
    2
  </th>
  <td>
    <strong>
      Team mock
    </strong>
  </td>
  <td
    className="highlight"
    key="0"
  >
    04:20 (1)
  </td>
  <td
    className="highlight"
    key="1"
  />
  <td
    className="highlight"
    key="2"
  />
  <td
    className="highlight"
    key="3"
  />
  <td>
    2
  </td>
</tr>
`;
